src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -89,4 +90,8 @@ public void addMessage(Message message) {
   public void setMessages(List<Message> messages) {
     this.messages = messages;
   }
+
+public int totalMessages() {
```

Comment:
Spacing here is kind of funky, let's fix that here and in the other methods added. Also it looks like you are going to want to set up eclipse to use 2 spaces instead of tabs because your changes are inconsistent with the rest of the codebase, sorry :D

Reply:
Fixed it & will commit!